modules/lib/lib-event/package.json

```diff
@@ -0,0 +1,53 @@
+{
+    "name": "lib-event",
+    "version": "0.1.0",
```

Comment:
what is this version? is it goint to be a version of types on npm? than it is hard to know what it works with XP 7.10

Reply:
`0.1.0` will be first version and meant to be used in other projects and tested. We may need to do additional changes before the actual release. The final version will follow the XP versioning.